app/src/main/res/drawable/ic_sync_problem.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="48dp"
+    android:height="48dp"
+    android:viewportWidth="48"
+    android:viewportHeight="48">
+  <path
+      android:fillColor="#FF000000"
```

Comment:
In our team we usually add this parameter to values/colors

Reply:
Like this `android:fillColor="@color/black"` or specific theme color?  Do I need to duplicate colors for this purpose in .kt and .xml files?

Reply:
Yes, you just add this color to values/color and just use it like @color/yourColor